### PR TITLE
Error on bad paths, remove unused parameter

### PR DESCRIPTION
**Pull request by ikonst**

### CODE REVIEW

File: xiblint/__main__.py [hunk 1]
```diff
@@ -58,22 +58,24 @@ def main():
...
     print_errors(errors, args.reporter)
 
     sys.exit(1 if errors else 0)
 
 
-def process_file(file_path, checkers, reporter):
-    _, ext = os.path.splitext(file_path)
-    if ext.lower() not in [u".storyboard", u".xib"]:
-        return []
+def process_file(file_path, checkers):
```
Comment (ikonst) [inline xiblint/__main__.py]: `reporter` argument was unused

File: xiblint/__main__.py [hunk 2]
```diff
@@ -58,22 +58,24 @@ def main():
...
-                for file_path in [os.path.join(root, file) for file in files]:
-                    checkers = config.checkers(file_path)
-                    errors += process_file(file_path, checkers, args.reporter)
+                for filename in files:
+                    if os.path.splitext(filename)[1].lower() in ('.storyboard', '.xib'):
+                        file_path = os.path.join(root, filename)
+                        checkers = config.checkers(file_path)
+                        errors += process_file(file_path, checkers)
+        else:
+            print("Error: Invalid path '{}'".format(path))
```
Comment (ikonst) [inline xiblint/__main__.py]: Now we won't silently error out if path does not exist (or doesn't end in .storyboard / .xib).

**Comment by ikonst** [comment]
^ @sunshinejr

File: xiblint/__main__.py (hunk 1 above)
Comment (sunshinejr) [reply xiblint/__main__.py]: Nice catch! Probably leftovers from moving things around between functions.

File: xiblint/__main__.py (hunk 2 above)
Comment (sunshinejr) [reply xiblint/__main__.py]: Awesome 👍

**Pull request merged**